packages/protocol/contracts/authorizers/interfaces/IRrpAuthorizerNew.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+interface IRrpAuthorizerNew {
```

Comment:
This is probably a temporary file, shouldn't this be renamed back to IRrpAuthorizer?

Reply:
It is temporary but updating `IRrpAuthorizer.sol` breaks the contracts in the `authorizers/`directory. I didn't want to do that because you're working on their tests.